scripts/make_api_rst.py

```diff
@@ -78,6 +79,73 @@ def ltr_tag(v):
     r"""^(?:([\w.]+::)?([\w.]+\.)?(\w+)\s*(?:\((.*)\)(?:\s*->\s*([\w.]+(?:\[.*?\])?))?(?:\s*\[(signal)\])?)?)?$"""
 )
 
+group_names = {
```

Comment:
to be coherent, I would move this to https://github.com/qgis/pyqgis-api-docs-builder/blob/main/pyqgis_conf.yml

Reply:
@3nids done